route/src/route_generator_worker.cpp

```diff
@@ -156,6 +173,7 @@ namespace route {
 
```

Comment:
Line 111 ROUTE_SELECTION needs to be updated to SELECTION.
Line 123 and 124 ROUTE_FAILURE needs to be "ROUTE_FAILED.